crates/kubelet/src/mio_uds_windows/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "mio-uds-windows"
+version = "0.1.0"
+license = "MIT"
+authors = ["Azure IoT Edge Devs"]
+description = "Unix Domain Sockets for mio...on Windows!"
+repository = "https://github.com/Azure/mio-uds-windows"
```

Comment:
if this is a permanent fork, should we update the crate info?

Reply:
I'm also wondering if this crate (along with any others introduced here) should be pulled out of Krustlet and maintained as a separate crate. They appear to help in some regards (file watching, Windows UDS, gRPC sockets...) but they're semi-orthogonal to Krustlet.

What do you think about pulling these crates out to be maintained separately?

Reply:
Oh! That cargo.toml file shouldn't even be there any more. I'll remove it

As for the separate crate question, ff we put them out into another crate, we'd have to publish them in order to publish krustlet as a crate. I don't want to accidentally put yet another fork of this out into the crates ecosystem when it is purely for use by us